addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/environment.html

```diff
@@ -93,7 +94,30 @@ <h3>Variables</h3>
 Variables can be defined in the 'vars' section.
 They can be hardcoded, refer to other variables or refer to system environment variables in the same way as above, e.g. '${envvar}'. 
 In case there are two variables with the same name, the value of the system variable would be preferred. 
-  
+
+<h3>Configs</h3>
+
+The 'configs' section can be used to define any value that can be set in the ZAP configuration file, which works in the same way as the ZAP '-config' command line parameter.
+<p>
+The advantages of using this section instead of the command line are:
+<ol>
+<li>The configuration will be self contained in the plan
+<li>The ZAP team will monitor they keys used in this way (when telemetry is enabled) and prioritise adding full support for the most frequently used one
+<li>Warnings will be raised when a key being used has a better solution available
+</ol>
+
+No validation is performed on the keys and values, so you are responsible for checking that your configuration
+is correct.
+<p>
+The recommended way to determine the keys is documented in the FAQ:
+<a href="https://www.zaproxy.org/faq/how-do-you-find-out-what-key-to-use-to-set-a-config-value-on-the-command-line/">How do you find out what key to use to set a config value on the command line?</a>
+<p>
+A working example, which sets the number of passive scan threads, is:
+<pre>
+configs:
+  pscans.threads: 8
+</pre> 
```

Comment:
Should we be putting together some automation to generate docs for the website?

Not that we want to encourage more use of this, but the arrays always raise questions/confusion. But it also isn't terribly user-friendly to have to install, use, and configure ZAP to dig up the details.